src/common/FileSystem.cpp

```diff
@@ -2666,7 +2684,23 @@ const PakInfo* FindPak(Str::StringRef name, Str::StringRef version)
 		return &*(iter - 1);
 }
 
-const PakInfo* FindPak(Str::StringRef name, Str::StringRef version, uint32_t checksum)
+const PakInfo* FindPak(Str::StringRef name, Str::StringRef version)
+{
+	const FS::PakInfo* pak = nullptr;
+
+	Cmd::Args pakprefixes(fs_pakprefixes.Get());
+	for (const std::string &pakprefix: pakprefixes)
+	{
+		pak = FS::FindPakNoPrefix(pakprefix + "/" + name, version);
```

Comment:
Is there a reason to force the prefix to include a slash?

If we do decide to force the `/`, I suggest using Path::Build so that it will still work if the user sets the cvar to `dev/` rather than `dev`.

Reply:
I simply forgot about `Path::Build`, now the code uses it.